core/src/components/textarea/textarea.ionic.scss

```diff
@@ -166,3 +174,41 @@ ion-icon {
 :host(.has-focus) .textarea-highlight {
   transform: scale(1);
 }
+
+// Textarea - Disabled
+// ----------------------------------------------------------------
+
+:host(.textarea-disabled) {
+  // color for the text within the textarea
+  --color: #{globals.$ion-primitives-neutral-500};
+  --background: #{globals.$ion-primitives-neutral-100};
+  --border-color: #{globals.$ion-primitives-neutral-300};
+  --placeholder-color: #{globals.$ion-primitives-neutral-500};
+
+  pointer-events: none;
+}
+
+:host(.textarea-disabled:not(.ion-valid)) .textarea-bottom .helper-text,
+:host(.textarea-disabled) .textarea-bottom .counter,
+:host(.textarea-disabled) .label-text-wrapper {
+  color: globals.$ion-primitives-neutral-500;
+}
+
+:host(.textarea-disabled.has-focus.ion-valid) {
+  --border-color: rgba(#{globals.$ion-semantics-success-base-rgb}, 0.6);
+}
+
+:host(.textarea-disabled.ion-touched.ion-invalid) {
+  --border-color: rgba(#{globals.$ion-semantics-danger-base-rgb}, 0.6);
+}
+
+:host(.textarea-disabled.ion-color) {
+  --border-color: #{current-color(base, 0.6)};
+}
+
+:host(.textarea-disabled.has-focus.ion-valid) .textarea-bottom .helper-text,
+:host(.textarea-disabled.ion-touched.ion-invalid) .error-text,
+:host(.textarea-disabled.ion-color) .textarea-bottom .helper-text,
+:host(.textarea-disabled.ion-color) .helper-text {
+  opacity: 0.6;
+}
```

Comment:
Where are these located in the designs? If these are assumptions then we should either exclude them because they are not in the designs or reach out to the design team.

Reply:
I've follow the input rules, but I'll remove it